mern/server/interactor/transcript_interactor.js

```diff
@@ -0,0 +1,45 @@
+import TranscriptDAO from "../DAO/transcript_dao.js";
+//Interactor calling the Transcript DAO functions after properly checking the inputs
+
+export default class TranscriptInteractor {
+  //Empty constructor
+  constructor(props) {
+    super(props);
+  }
+
+  //Gets transcript by ID
+  getTranscript(req, res) {
+    TranscriptDAO.getTranscriptByID(req, res);
+  }
+
+  //Creates transcript and checks success of creation
+  createTranscript(req, res) {
```

Comment:
This function needs to be changed, as the same get function also needs to be changed in flow and qa.

Check if the req.data exists, and throws error if fields don't exists.